### PR TITLE
Update the Administration Console description in the Administration Guide

### PR DESCRIPTION
**Pull request by avpinchuk**
* Delete irrelevant Windows info.
* Add the Administration Console startup options description.

### CODE REVIEW

**Comment by avpinchuk** [comment]
@OndroMih, please review.

**Review by dmatej — APPROVED** [review]

File: docs/administration-guide/src/main/asciidoc/overview.adoc [hunk 1]
```diff
@@ -608,6 +599,100 @@ Security]" in {productName} Security Guide.
 ====
 
 
+You can change the Administration Console startup behavior using the
+startup options.
+
+Available startup options are:
+
+`default`::
+The Administration Console loads on demand the first access it. This is a default option.
```
Comment (OndroMih) [inline docs/administration-guide/src/main/asciidoc/overview.adoc:608]:
```suggestion
The Administration Console loads on demand when its first accessed. This is the default option.
```

File: docs/administration-guide/src/main/asciidoc/overview.adoc [hunk 2]
```diff
@@ -608,6 +599,100 @@ Security]" in {productName} Security Guide.
...
+Enabling on demand loading of the Administration Console involves changing the value of the
+property `server.admin-service.property.adminConsoleStartup`. This property controls the
+Administration Console loading.
+
+1. Ensure that the DAS is running.
++
+2. Change the value of the `server.admin-service.property.adminConsoleStartup` property to `default`.
+You can change this value either by setting a configuration property or by editing a file.
+* To change this value by setting a configuration property, set the configuration property
+`server.admin-service.property.adminConsoleStartup` to a value `default`.
```
Comment (OndroMih) [inline docs/administration-guide/src/main/asciidoc/overview.adoc:630]:
```suggestion
`server.admin-service.property.adminConsoleStartup` to the value `default`.
```

File: docs/administration-guide/src/main/asciidoc/overview.adoc [hunk 3]
```diff
@@ -608,6 +599,100 @@ Security]" in {productName} Security Guide.
...
+[source]
+----
+asadmin> set server.admin-service.property.adminConsoleStartup=never
+----
+
+3. Restart the DAS.
+For instructions, see xref:domains.adoc#to-restart-a-domain[To Restart a Domain].
+
+
+You can display the help material for a page in the Administration
```
Comment (OndroMih) [inline docs/administration-guide/src/main/asciidoc/overview.adoc:689]:
```suggestion
##### Help pages in the Administration Console

You can display the help material for a page in the Administration
```

**Review by OndroMih — CHANGES_REQUESTED** [review]

**Review by arjantijms — APPROVED** [review]

**Pull request merged**